package.json

```diff
@@ -5,10 +5,9 @@
 	"main": "server.js",
 	"type": "module",
 	"scripts": {
-		"start": "nodemon src/server.js",
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"start": "nodemon src/server.js"
```

Comment:
Do we better add "build": "tsc" and "test": "jest" here, so we do not need to merge later?

Reply:
I have made the changes here